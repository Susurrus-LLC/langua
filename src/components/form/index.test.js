import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Form from '.'

describe('Form', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Form />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders accurately with and without props and children', () => {
    let tree = renderer.create(<Form />).toJSON()
    expect(tree).toMatchSnapshot()

    tree = renderer.create(<Form name='test'>Test</Form>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
