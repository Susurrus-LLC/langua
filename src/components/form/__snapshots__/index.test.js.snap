// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders accurately with and without props and children 1`] = `
<form
  className="form"
/>
`;

exports[`Form renders accurately with and without props and children 2`] = `
<form
  className="form"
  name="test"
>
  Test
</form>
`;
